src/main.c

```diff
@@ -391,6 +393,7 @@ int main(int argc, char* args[])
     {"version", 'v', OPTPARSE_NONE},
     {"record", 'r', OPTPARSE_OPTIONAL},
     {"scale", 's', OPTPARSE_REQUIRED},
+    {"wren2cstring", 'w', OPTPARSE_REQUIRED},
```

Comment:
I'm not certain `OPTPARSE_REQUIRED` is correct because we don't expect an argument for this flag. `OPTPARSE_NONE` makes more sense.

Reply:
Yes we expect at least the filename to be encoded :)

Reply:
That is true, but we aren't using optparse here to retrieve that argument.

Reply:
Is useful to check it before calling the function by optparse validation. Is a way to check before executing :)

Reply:
What happens if no other arguments are provided? `dome -e`

Reply:
./dome: option requires an argument -- 'e'